regulations/static/regulations/js/unittests/specs/views/comment-view-spec.js

```diff
@@ -35,7 +36,15 @@ describe('CommentView', function() {
         '</form>' +
       '</div>'
     );
-    $(document.body).empty().append($el);
+    var $template = $(
+      '<script id="comment-attachment-template" type="text/template">' +
+        '<div></div>' +
+      '</script>'
+    );
+    var $body = $(document.body)
+      .empty()
+      .append($el)
+      .append($template);
```

Comment:
Shouldn't be any harm to it, since this is just the test setup--there's nothing else in the body, and we want to clear out junk left behind by previous test runs. I can move the fixtures into a #fixtures element instead if it would be clearer.